docs/documentation/development/bindings/categories.md

```diff
@@ -14,7 +14,44 @@ The thing type definition allows to specify a category. User interfaces can pars
 
 | Category        | Description                                          |
 |-----------------|------------------------------------------------------|
-
+| Aircondition    | Air condition devices incl. Fans |
+| BinarySensor    | General purpose sensor that returns a boolean value, i.e. a contact that reports open/close on a door or window |
+| Bridge          | Bridges/Gateway need to access other devices like used by Philips Hue for example |
```

Comment:
Might ["NetworkAppliance"](https://www.google.de/search?q=network+appliance&tbm=isch) be a good, more general category?